packages/react-next/src/components/Link/Link.types.ts

```diff
@@ -94,3 +91,20 @@ export interface ILinkStyleProps {
 export interface ILinkStyles {
   root: IStyle;
 }
+
+/**
+ * {@docCategory Link}
+ */
+export interface ILinkSlots {}
```

Comment:
Because `Link` is a primitive and does not have slots, I don't think this is used anywhere.